continuousflex/protocols.conf

```diff
@@ -90,3 +90,17 @@ TomoFlow = [
 	{"tag": "section", "text": "5. 3D averaging & animations", "children": [
 	    {"tag": "protocol", "value": "FlexProtDimredHeteroFlow", "text": "Clustering and trajectories"}
 	]}]
+
+Genesis = [
+	{"tag": "section", "text": "1. Import atomic model", "children": [
+	    {"tag": "protocol", "value": "ProtImportPdb", "text": " Input PDB", "icon": "bookmark.png"}
+	]},
+		{"tag": "section", "text": "2. Import target EM data", "children": [
+        {"tag": "protocol", "value": "ProtImportVolumes", "text": "Input volume", "icon": "bookmark.png"}
+	]},
```

Comment:
In which one of these two we use NMMD? I understand that the protocol is for Genesis in general, but since this tree is for NMMD, you can say something like "NMMD using Genesis"?
Also, Don't you think it is better to use "Flexible fitting using Molecular Dynamics and Normal Mode Analysis" instead of Molecular Dynamics alone?